build/packages.targets

```diff
@@ -0,0 +1,99 @@
+<Project>
+    <PropertyGroup>
+        <MicrosoftBuildPackageVersion Condition="'$(MicrosoftBuildPackageVersion)' == ''">16.5.0-preview-19606-01</MicrosoftBuildPackageVersion>
+        <NewtonsoftJsonPackageVersion Condition="$(NewtonsoftJsonPackageVersion) == ''">9.0.1</NewtonsoftJsonPackageVersion>
```

Comment:
Doesn't need to be in this PR, but I think we can consider upgrading Newtonsoft.Json now. In the past I believe we kept it at this version because the .NET Core SDK used it, but they've since upgraded, so I think we should too. It might let us get rid of some NU1605 NoWarns

Reply:
I think that's fair.